api/tests/acceptance/application/organizations/organization-controller-import-students_test.js

```diff
@@ -0,0 +1,668 @@
+              '<PRENOM3></PRENOM3>' +
+              '<DATE_NAISS>01/07/1994</DATE_NAISS>' +
+              '<CODE_PAYS>100</CODE_PAYS>' +
+              '<CODE_DEPARTEMENT_NAISS>033</CODE_DEPARTEMENT_NAISS>' +
+              '<CODE_COMMUNE_INSEE_NAISS>33318</CODE_COMMUNE_INSEE_NAISS>' +
+              '<CODE_MEF>12341234</CODE_MEF>' +
+              '<CODE_STATUT>ST</CODE_STATUT>' +
+            '</ELEVE>';
+
+          const bufferWithMalformedStudent = iconv.encode(
+            '<?xml version="1.0" encoding="ISO-8859-15"?>' +
+            '<BEE_ELEVES VERSION="2.1">' +
+              '<DONNEES>' +
+                '<ELEVES>' +
+                  studentThatCantBeUpdatedBecauseBirthdateIsMissing +
+                  studentThatCouldBeUpdated +
+                '</ELEVES>' +
+                '<STRUCTURES>' +
+                  '<STRUCTURES_ELEVE ELEVE_ID="0001">' +
+                    '<STRUCTURE>' +
+                      '<CODE_STRUCTURE>4A</CODE_STRUCTURE>' +
+                      '<TYPE_STRUCTURE>D</TYPE_STRUCTURE>' +
+                    '</STRUCTURE>' +
+                  '</STRUCTURES_ELEVE>' +
+                  '<STRUCTURES_ELEVE ELEVE_ID="0002">' +
+                    '<STRUCTURE>' +
+                      '<CODE_STRUCTURE>4A</CODE_STRUCTURE>' +
+                      '<TYPE_STRUCTURE>D</TYPE_STRUCTURE>' +
+                    '</STRUCTURE>' +
+                  '</STRUCTURES_ELEVE>' +
+                  '<STRUCTURES_ELEVE ELEVE_ID="0003">' +
+                    '<STRUCTURE>' +
+                      '<CODE_STRUCTURE>4A</CODE_STRUCTURE>' +
+                      '<TYPE_STRUCTURE>D</TYPE_STRUCTURE>' +
+                    '</STRUCTURE>' +
+                  '</STRUCTURES_ELEVE>' +
+                '</STRUCTURES>' +
+              '</DONNEES>' +
+            '</BEE_ELEVES>', 'ISO-8859-15');
+
+          options.payload = bufferWithMalformedStudent;
+
+          databaseBuilder.factory.buildStudent({
+            lastName: 'LALOUX',
+            firstName: 'RENE',
+            nationalStudentId: '123',
+            organizationId
+          });
+          databaseBuilder.factory.buildStudent({
+            lastName: 'UEMATSU',
+            firstName: 'NOBUO',
+            nationalStudentId: '456',
+            organizationId
+          });
+
+          await databaseBuilder.commit();
+        });
+
+        it('should not update any student and return a 409 - Conflict', async () => {
+          // when
+          const response = await server.inject(options);
+
+          // then
+          const students = await knex('students').where({ organizationId });
+          expect(_.map(students, 'lastName')).to.have.members(['LALOUX', 'UEMATSU']);
+          expect(response.statusCode).to.equal(409);
+          expect(response.result.errors[0].detail).to.equal('L\'enregistrement des élèves a rencontré une erreur.');
+        });
+      });
+
+      context('when a student cant be updated but another could be created', () => {
```

Comment:
ce test passe toujours si dans le usecase tu inverses la création et la mise à jour ?

Reply:
Vu avec @lisequesnel @VincentHardouin @Yuhiba @jonathanperret 

On a rajouté un test 'when a student cant be created but another could be updated', qui ne passe pas actuellement.
Une méthode de repo va être ajoutée pour avoir le comportement attendu.